src/App.css

```diff
@@ -307,16 +329,30 @@
   max-width: 85rem;
 }
 
+.comments button {
+  border: none;
+  background: transparent;
+  color: var(--blue0);
+}
+
+.comments button:hover {
+  border: 0.01rem solid var(--blue0);
+  background: transparent;
+}
```

Comment:
* Well done with the design but there are still some differences between your design and the given template things as spacing, alignments, and  user experience kindly try to make your design look like the given one below 👇 are some highlighted parts to consider  
* ## YOURS 
![image](https://github.com/Amazinggracee/Bookstore/assets/52497246/01e23d8a-1daa-4ff7-a392-9bf6d3cfed46)
![image](https://github.com/Amazinggracee/Bookstore/assets/52497246/5c1c27b3-3d75-4c94-ba78-6706ee864e7e)

## GIVEN DESIGN 
![image](https://github.com/Amazinggracee/Bookstore/assets/52497246/62c179de-99eb-4355-a2fa-f229053154c6)